products/urls.py

```diff
@@ -0,0 +1,8 @@
+from django.urls import path
+
+from products.views import ProductListView
+
+urlpatterns = {
+    path('', ProductListView.as_view()),
+    path('/<int:cat_id>', ProductListView.as_view()),
```

Comment:
```suggestion
    path('/<int:category_id>', ProductListView.as_view()),
```
`category_id`라고 정확하게 명시해주는건 어떨까요?

Reply:
넵